libgalois/include/katana/PropertyGraph.h

```diff
@@ -408,6 +408,16 @@ class KATANA_EXPORT PropertyGraph {
   PropertyGraph(katana::GraphTopology&& topo_to_assign) noexcept
       : rdg_(), file_(), topology_(std::move(topo_to_assign)) {}
 
+  PropertyGraph(
+      katana::GraphTopology&& topo_to_assign,
+      NUMAArray<EntityTypeID> node_entity_type_id,
```

Comment:
I recommend putting `&&` on moveable objects (or objects to be moved).

Reply:
Yup, good call. The copy constructor for `NUMAArray` is deleted, so I don't think we can get in too much trouble. But I agree that `&&` is an important readability improvement.